binplace.targets

```diff
@@ -88,17 +87,35 @@
     </Copy>
   </Target>
 
-  <Target Name="BinPlaceRuntimeAssembly" DependsOnTargets="GetBinplaceDirs;GetBinplaceItems">
-    <Copy Condition="'@(RuntimeDir)' != ''"
-          SourceFiles="@(BinplaceItem)"
-          DestinationFolder="%(RuntimeDir.Identity)"
-          SkipUnchangedFiles="true"
-          OverwriteReadOnlyFiles="true"
-          Retries="$(CopyRetryCount)"
-          RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
-          UseHardlinksIfPossible="$(BinplaceUseHardlinksIfPossible)">
-      <Output TaskParameter="DestinationFiles" ItemName="FileWrites" />
-    </Copy>
+  <UsingTask TaskName="SaveItems" AssemblyFile="$(CoreFxToolsTaskDir)CoreFx.Tools.dll"/>
+  <Target Name="BinplaceProps"
+          Condition="'@(PackageFileDir)' != ''"
+          DependsOnTargets="GetBinplaceItems" >
+    <PropertyGroup>
+      <_propsFilename>$(TargetName)</_propsFilename>
+      <_propsFilename Condition="'$(_propsFilename)' == ''">$(MSBuildProjectName)</_propsFilename>
+      <_projectDirLength>$(ProjectDir.Length)</_projectDirLength>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <_itemsToSave Include="@(BinplaceItem)">
+        <!-- intentionally empty: to be set by pkgproj -->
+        <TargetPath></TargetPath>
+      </_itemsToSave>
+      
+      <!-- Include source files. -->
+      <_itemsToSave Condition="'@(Compile)' != ''" Include="@(Compile->'%(FullPath)')">
+        <TargetPath>src</TargetPath>
+        <TargetPath Condition="$([System.String]::Copy('%(FullPath)').StartsWith('$(ProjectDir)'))">src/$([System.String]::Copy('%(FullPath)').Substring($(_projectDirLength)).Replace('\', '/'))</TargetPath>
+        <IsSourceCodeFile>true</IsSourceCodeFile>
+      </_itemsToSave>
+    </ItemGroup>
+    
+    <Message Importance="low" Text="PackageFileDir: @(PackageFileDir)" />
+
+    <SaveItems ItemName="FileToPackage"
+               Items="@(_itemsToSave)"
+               Files="@(PackageFileDir->'%(Identity)\$(_propsFilename).props')" />
```

Comment:
Not packaged.  They just get imported by the pkgrpojs to create the file items (rather than a wildcard in the pkgproj).